preprocessing/nextclade/src/loculus_preprocessing/backend.py

```diff
@@ -77,7 +77,11 @@ def parse_ndjson(ndjson_data: str) -> Sequence[UnprocessedEntry]:
             continue
         # Loculus currently cannot handle non-breaking spaces.
         json_str_processed = json_str.replace("\N{NO-BREAK SPACE}", " ")
-        json_object = json.loads(json_str_processed)
+        try:
+            json_object = json.loads(json_str_processed)
+        except json.JSONDecodeError as e:
+            error_msg = f"Failed to parse JSON: {json_str_processed}"
+            raise Exception(error_msg) from e
```

Comment:
We might actually want to catch this and not crash prepro as a result, just log an error - it's a fault of the backend, not a fault in the prepro pipeline.